src/main/java/Task.java

```diff
@@ -0,0 +1,45 @@
+
+/**
+ * Each task will hold the status of done or not done, plus description of the
+ * task to be done.
+ */
+public class Task {
+    private String description;
+    private boolean isDone;
```

Comment:
Good naming of this variable, it is clear that it is a boolean variable just from the name! 👍